src/Utilities/Utilities.jl

```diff
@@ -26,6 +26,9 @@ function print_with_acronym(io::IO, s::AbstractString)
     print(io, s)
 end
 
+
+function _reverse_dict end
```

Comment:
This could probably use a docstring explaining what it does.

Reply:
done